frontend/app-development/features/administration/components/LegacyAdministration.module.css

```diff
@@ -0,0 +1,35 @@
+.avatar {
+  max-height: '2em';
+}
+.sidebarHeader {
+  margin-bottom: 20;
+  font-size: 20;
+  font-weight: 500;
+}
+
+.sidebarHeaderSecond {
+  margin-top: 36;
+}
+
+.sidebarInfoText {
+  font-size: 16;
+  margin-bottom: 12;
+}
+
+.iconStyling {
+  font-size: 35;
+  text-align: 'right';
+}
+
+.sidebarServiceOwner {
+  margin-top: 10;
+}
+
+.sidebarCreatedBy {
+  font-size: 16;
+  margin-top: 10;
+}
+
+.margin-bottom_24 {
```

Comment:
I can't see where the class name margin-bottom_24 is used. I suggest we remove it, agree? 😊

Reply:
I checked each class and none of them was actually used, so I deleted the file